### PR TITLE
fix: update sort params to correctly sort

### PR DESCRIPTION
**Pull request by rtirserio**
sort params weren't sorting in the right direction or in some cases the correct field

### CODE REVIEW

File: src/Constants/Sort.js [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-
 export const POSITION_SEARCH_SORTS = {
   options: [
     { value: '', text: 'Sort option', disabled: true },
     { value: 'position__title', text: 'Position title: A-Z' },
-    { value: '-position__grade', text: 'Grade: Low to high' }, // sort by grade "ranking"
-    { value: '-position__bureau', text: 'Bureau: A-Z' }, // numbers first, then A-Z
-    { value: `-${COMMON_PROPERTIES.posted}`, text: 'Posted date: Most recent' }, // sort by soonest posted_date
-    { value: 'position__current_assignment__estimated_end_date', text: 'TED: Soonest' },
+    { value: 'position__grade', text: 'Grade: Low to high' }, // sort by grade "ranking"
```
Comment (mjoyce91) [inline src/Constants/Sort.js]: This should actually be -position__grade, as "06" is logically the "lowest" grade.

File: src/Constants/Sort.js [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+    { value: 'position__bureau', text: 'Bureau: A-Z' }, // numbers first, then A-Z
```
Comment (mjoyce91) [inline src/Constants/Sort.js]: I'm getting W's first for this one

**Comment by mjoyce91** [comment]
The direction of all of these is backwards on my end

**Comment by rtirserio** [comment]
@mjoyce91 fixed

**Pull request merged**